boards/arm/da1469x_dk_pro/doc/index.rst

```diff
@@ -0,0 +1,119 @@
+.. _da1469x_dk_pro:
+
+DA1469x Development Kit Pro
+###########################
+
+Overview
+********
+
+The DA1469x Development Kit Pro hardware provides support for the Renesas
+DA1469x ARM Cortex-M33 MCU family. The development kit consist of a motherboard
+with connectors and integrated debugger and an interchangeable daughterboard
+with an actual MCU (e.g. DA14695 or DA14699).
+
+.. figure:: da14695-00hqdevkt-board.jpg
+     :width: 442px
+     :align: center
+     :alt: DA14695 Development Kit Pro
+
+     DA14695 Development Kit Pro (Credit: Renesas Electronics Corporation)
+
+Hardware
+********
+
+DA1469x Development Kit Pro has two external oscillators. The frequency of
+the sleep clock is 32768 Hz. The frequency of the system clock is 32 MHz.
+
+Supported Features
+==================
+
+The _da1469x_dk_pro board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
```

Comment:
Where is WDT driver?
Why not mention pinctrl driver here?

Reply:
Watchdog is only initialized by SoC code, there's no dedicated driver (yet) - I'll just remove entry for now.
I don't really see pinctrl driver mentioned in any of board docs...